### ISSUE TITLE
[Feature Request] PWA

### ISSUE THREAD

**Issue by heberq1978** [FOSSBilling/FOSSBilling#695]
There would be a way to insert a PWA system where we can send push, invoices and tickets. Because more than 99% of customers do not use the customer panel.

Alternative free push: https://onesignal.com/

Thank You
Heber Queiroz - BR

**Comment by BelleNottelling** [comment]
So, this is as an alternative to email, right?

**Comment by tobsowo** [comment]
Progressive Web Apps will be cool to have as an alternative to mobile app. More information can be found on [pwa.dev](https://pwa.dev).

**Comment by BelleNottelling** [comment]
> Progressive Web Apps will be cool to have as an alternative to mobile app. More information can be found on [pwa.dev](https://web.dev/progressive-web-apps).

Yeah, I know what a progressive web app is, but that doesn't seem to be what this issue is requesting

**Comment by heberq1978** [comment]
Come on, what happens a lot here in Brazil is that customers don't access the customer panel. As the PWA is a system aimed not only at mobile, but also at PC, it would be nice to have it, because with that it would force the client to have an application where he would not be able to say that it is not being formed, because the PWA offers several alternatives in communication issue.
One of the alternatives would be the Push issue that I gave as an example, but there are people here who created a PWA that was very complete, with a button for payment, history, ticket, services and many other resources that are also available in the client's panel.
In the case of this PWA, which I said is very complete, it's for WHMCS.

**Comment by John-S4** [comment]
> Come on, what happens a lot here in Brazil is that customers don't access the customer panel.

But that is their choice. If a client is receiving emails with notifications that they need to make a payment or do whatever else, and ignore them without clicking a link to go to the control panel to do it, what difference is there if they receive a push notification and ignore that. 

I'm not dismissing the suggestion, I just think that either clients respond to notifications or they don't and the delivery medium of notification isn't the deciding factor.

**Comment by admdly** [comment]
This may have been a product of its time - if this is still something desired and there are more details available please reopen this issue.
